include/jitk/statistics.hpp

```diff
@@ -71,13 +72,13 @@ class Statistics {
     std::chrono::duration<double> time_copy2dev{0};
     std::chrono::duration<double> time_copy2host{0};
     std::chrono::duration<double> time_ext_method{0};
-    std::map<std::string, std::pair<int, std::chrono::duration<double> > > time_per_kernel;
+    std::map<std::string, std::pair<uint64_t, std::chrono::duration<double> > > time_per_kernel;
```

Comment:
Could we have a comment on this, what the `string` and `uint64_t` are? I mean, just write that the keys are the kernel names and the pair is times executed and duration.

Reply:
This doesn't exist any longer - I now put the kernel statistics into a struct instead of the pair. But I will add a comment that the key is the kernel filename.